files/image_config/config-setup/config-setup

```diff
@@ -56,17 +57,31 @@ usage()
 EOF
 }
 
+# Factory command usage and help
+usage_factory()
```

Comment:
>usage_factory [](http://example.com/codeflow?start=0&length=13)

Could you merge this function to above usage()?

Reply:
Thanks for the comments.

usage_factory() is used for "config-setup factory" help command.
while useage() is used for "config-setup" help command.
Do you mean by merge, to combine the two functions into one function with parameter ? 
if so, why do you think it is necessary ?